src/test/java/teammates/test/cases/webapi/GetFeedbackQuestionRecipientsActionTest.java

```diff
@@ -1,15 +1,46 @@
 package teammates.test.cases.webapi;
 
+import org.apache.http.HttpStatus;
+import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.Test;
 
+import teammates.common.datatransfer.DataBundle;
+import teammates.common.datatransfer.attributes.FeedbackQuestionAttributes;
+import teammates.common.datatransfer.attributes.FeedbackSessionAttributes;
+import teammates.common.datatransfer.attributes.InstructorAttributes;
+import teammates.common.datatransfer.attributes.StudentAttributes;
 import teammates.common.util.Const;
+import teammates.common.util.StringHelper;
 import teammates.ui.webapi.action.GetFeedbackQuestionRecipientsAction;
+import teammates.ui.webapi.action.Intent;
+import teammates.ui.webapi.action.JsonResult;
+import teammates.ui.webapi.output.FeedbackQuestionRecipientsData;
 
 /**
  * SUT: {@link GetFeedbackQuestionRecipientsAction}.
  */
 public class GetFeedbackQuestionRecipientsActionTest extends BaseActionTest<GetFeedbackQuestionRecipientsAction> {
 
+    private FeedbackSessionAttributes firstSessionInCourse1;
+    private FeedbackSessionAttributes secondSessionInCourse1;
+    private FeedbackSessionAttributes firstSessionInCourse2;
+    private StudentAttributes student1InCourse1;
+    private InstructorAttributes instructor1OfCourse1;
+    private InstructorAttributes instructor1OfCourse2;
+
+    @Override
+    @BeforeMethod
+    public void beforeTestMethodSetup() {
+        DataBundle testData = loadDataBundle("/FeedbackQuestionRecipientsTest.json");
```

Comment:
- `FeedbackQuestionRecipientsTest` -> `GetFeedbackQuestionRecipientsActionTest.json`
- If you don't need typicalDatabundle anymore. You should override `prepareTestData` so `typicalDatabundle` won't persist during **this** class.

Reply:
Actually I need the `typical bundle` in [testAccessControl](https://github.com/TEAMMATES/teammates/blob/6be738a64bb8bcec67c988930d81a8fb7744a5be/src/test/java/teammates/test/cases/webapi/GetFeedbackQuestionRecipientsActionTest.java#L171). Does it affect the persistence?

Reply:
I don't think so.